common/lib/src/realm_types.dart

```diff
@@ -79,8 +79,18 @@ class RealmStateError extends StateError implements RealmError {
   RealmStateError(String message) : super(message);
 }
 
+/// An error type that describes a client reset error condition.
+/// {@category Sync}
+class ClientResetError extends SessionError {
+  ClientResetError(
+    String message,
+    SyncErrorCategory category, {
+    bool isFatal = false,
```

Comment:
Isn't a client reset error always fatal?

Reply:
How can we check it?